Blazorise.slnf

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Do we really need to have `.slnf` files? Maybe add them to gitignore and just leave them locally?

Reply:
We don't if you don't like them.